src/com/untamedears/JukeAlert/util/OnlineGroupMembers.java

```diff
@@ -148,7 +149,7 @@ private Player getNextModerator() {
             GroupPermission perm = manager_.getPermissionforGroup(g);
             List<OfflinePlayer> mods = new ArrayList<OfflinePlayer>();
             for (UUID uuid: uuids)
-            	if (perm.isAccessible(g.getPlayerType(uuid), PermissionType.BLOCKS) && !g.isOwner(uuid))
+            	if (g.getPlayerType(uuid) != PlayerType.MEMBERS && !g.isOwner(uuid))
```

Comment:
This is a bit difficult. This class has a really weird system to iterate over players of a group and I didnt want to completly redo it. It just had hardcoded to sort the members of a group by who has BLOCKS permissions and who hasnt. It will only try this for members of the group though, so the person checked for will never be NOT_BLACKLISTED. This class has no actual use other than getting all members of a class in a very complicated way and supporting some legacy features, which we are not using. I decided against ripping the whole thing out and this band-aid fix, which should work. The actual permission check is somewhere completly else anyway.